CMakeLists.txt

```diff
@@ -123,6 +127,11 @@ set(CMAKE_CXX_STANDARD 17)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 set(CMAKE_CXX_EXTENSIONS OFF)
 
+# PIC for shared mode
+if(BUILD_SHARED_LIB)
```

Comment:
Seems like there's a typo in the BUILD_SHARED_LIB(s) name here. Regardless, I don't think this is necessary, CMAKE_POSITION_INDEPENDENT_CODE is set to ON by default for SHARED lib builds.

Reply:
* Typo fixed, added a more suggestive comment to it.
* It is needed in SHARED mode to force down **all static (externals)** to pickup the -fPIC.

Reply:
* Removed conditional.

Q: Do you want libslang.a ```STATIC``` variant to be PIC (for more extern linkae usability) ?